docs/development/codeguide.rst

```diff
@@ -134,10 +136,35 @@ Coding Style/Conventions
   using only 4 spaces for indentation, and never tabs.
 
 * Our testing infrastructure currently enforces a subset of the PEP8 style
-  guide. You can check locally whether your changes have followed these by
-  running the following `tox <https://tox.readthedocs.io/>`__ command::
+  guide, and some packages enforce stronger styling checks such as using
```

Comment:
The "some packages" is confusing to me. The `pre-commit.yaml` file doesn't indicate any exclusions, and astropy imports are not currently sorted in general. So if this PR is committed and developers install pre-commit, will the import order started getting fixed on a per-file basis in new PR's?

Reply:
No, pre-commit just runs our `isort` configuration, which has per-subpackage configuration options:
https://github.com/astropy/astropy/blob/c900c5bf700df032a99dfd4a8306e01cbaeb40dd/pyproject.toml#L35-L58

Reply:
Unrelated. But that's a good reminder we should continue ``isort``ing Astropy!